azure-pipelines.yml

```
@@ -77,6 +77,14 @@ steps:
     cargo test --verbose --all --features "test unstable"
   displayName: test
 
+# TODO: Use the latest version of tarpaulin once our code compiles with it
+- script: |
+    cargo install --verbose cargo-tarpaulin --version '0.13.3'
+    cargo tarpaulin --verbose --all --features "test unstable" --out Xml --run-types  Tests Doctests
+    bash <(curl -s https://codecov.io/bash)
+  displayName: code_coverage
+  condition: and( eq( variables['Agent.OS'], 'Linux' ), eq( variables['rustup_toolchain'], 'stable' ) )
```

```suggestion
  condition: and( eq( variables['Agent.OS'], 'Linux' ), eq( variables['rustup_toolchain'], 'nightly' ) )
```

If you check the output of the code_coverage task on Azure DevOps, it looks like the doctests are still not running. I looked into it a bit and it seems like nightly is needed for that. Let's try this change and see if that fixes it. I think the coverage should be at around ~60% once we're done. (It's at 51.13% right now.)

---

Yes indeed, I'll add it. Testing on my machine yields 61.06% coverage on this branch.